dotCMS/src/main/java/com/dotmarketing/filters/CMSFilter.java

```diff
@@ -380,5 +348,11 @@ private String xssCheck(String uri, String queryString) throws ServletException
         return rewrite;
     }
 
+    @Override
+    public void init(FilterConfig arg0) throws ServletException {
+        // TODO Auto-generated method stub
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)